File: Basic Coding Problems/02 If - Elses/033 Write a program to input all sides of a triangle and check whether triangle is valid or not/Java/triangle_validation.java
import java.util.Scanner;

public class triangle_validation {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the three sides of a triangle: ");
        int sideA = scanner.nextInt();
        int sideB = scanner.nextInt();
        int sideC = scanner.nextInt();

        if (sideA + sideB > sideC && sideA + sideC > sideB && sideB + sideC > sideA) {
            System.out.println("The triangle is valid.");
        } else {
            System.out.println("The triangle is not valid.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/033 Write a program to input all sides of a triangle and check whether triangle is valid or not/C/triangle_validation.c
#include <stdio.h>

int main() {
    int side_a, side_b, side_c;

    printf("Enter the three sides of a triangle: ");
    scanf("%d %d %d", &side_a, &side_b, &side_c);

    if (side_a + side_b > side_c && side_a + side_c > side_b && side_b + side_c > side_a) {
        printf("The triangle is valid.");
    } else {
        printf("The triangle is not valid.");
    }

    return 0;
}